From ad4e26a2e45b322434bd1fd1ecc77ec9f2ac7a97 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Tue, 23 Jun 2015 15:37:27 +0200
Subject: [PATCH] sqlite3.js: hardcode node_sqlite3.node path

* trying to find it, doesn't work correctly when cross compiling

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 lib/sqlite3.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/sqlite3.js b/lib/sqlite3.js
index 466b902..6464d87 100644
--- a/lib/sqlite3.js
+++ b/lib/sqlite3.js
@@ -1,6 +1,6 @@
 var binary = require('node-pre-gyp');
 var path = require('path');
-var binding_path = binary.find(path.resolve(path.join(__dirname,'../package.json')));
+var binding_path = '/usr/lib/nodejs/node_sqlite3.node';
 var binding = require(binding_path);
 var sqlite3 = module.exports = exports = binding;
 var EventEmitter = require('events').EventEmitter;
-- 
2.4.4

